### ISSUE TITLE
'dotnet add package --info' incorrectly suggests adding multiple package references at once

### ISSUE THREAD

**Issue by DustinCampbell** [dotnet/cli#5388]
## Steps to reproduce
First, run `dotnet add package --help` and note that it offers the following help for `[args]`:

```
Additional Arguments:
 Package references to add
```

Next, try actually adding multiple references.

```
dotnet add package --version 2.0.0-rc2 Microsoft.CodeAnalysis.CSharp.Features Microsoft.CodeAnalysis.CSharp.Workspaces
```

## Expected  behavior
Either it works or the `--help` doesn't suggest this.

## Actual behavior
```
Please specify one package reference to add.
```

## Environment data
`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-rc4-004530)

Product Information:
 Version:            1.0.0-rc4-004530
 Commit SHA-1 hash:  4f0b7bd58c

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15007
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.0-rc4-004530
```

**Comment by billwert** [comment]
looks like #5740 covered this?

**Comment by mairaw** [comment]
And I did a PR to fix this: https://github.com/dotnet/cli/pull/5875 
I didn't know there were issues for it already.

**Comment by mairaw** [comment]
And @spboyer too! We were working on the same docs. I see this is no longer an issue when I merged master into my PR. 😄

**Comment by mairaw** [comment]
This was resolved with PR #5741. Closing.
